communities/edinburgh-tech-learners/impact-proposal.md

```diff
@@ -0,0 +1,121 @@
+
+### Goals
+
+Edinburgh has a lot of tech events aimed at beginners, and a lot that assume a more advanced audience. There’s little in between — once somebody has reached the end of the beginner content, it’s not clear what they should do next. My goal is to bring this community together, and provide a space where they can learn, socialise, and develop their skills together. My proposal will address this goal by providing an inaugural event for this community, as well as an online space for ongoing community participation.
+
+### Communication
+
+As identified above, Facebook is likely to be the most effective way for the members of the community to engage with events and communicate with each other. However, I think it’s important to ensure that people aren’t forced to use Facebook just to take part in that community, so to that end essential information will be available through a website or mailing list, ideally with a subscribable calendar feed.
+
+## Your Proposal’s Impact
+
+### Target audience
+
+#### Who are you expecting to impact based on your proposal?
+
+As above, I plan to impact people (mostly students of the University of Edinburgh) who are underserved by the communities that aim at either the beginner or advanced level.
+
+### Community needs
+
+#### What specific issue are you addressing with your proposal?
+
+I am addressing the lack of a tech community in Edinburgh aimed at helping those of intermediate skill level to develop and move on to the advanced-level communities.
+
+### Resources
+
+#### What resources do you need to accomplish your proposal?
+
+Very little! All I really need is a room where I can hold events. There are other small things I could provide — snacks, stickers, etc.
+
+#### How will you obtain these resources?
+
+By collaborating with CompSoc, the university's computing and artificial intelligence society, I will be able to book access to university venues for little or no cost. In future, I could also ask tech companies in Edinburgh to allow us to meet in their offices. The other little niceties could likely be sourced from Edinburgh tech companies by providing the opportunity to provide a sponsorship at a very low cost.
+
+### Media
+
+#### How are you preparing to deliver your community impact proposal?
+
+I will get in touch with the various groups I'd like to use to find members of the community, and ask if I can reach out to their members somehow — this would preferably be me talking about my event at a gathering of that community in person, but could also take the form of me making a post in an online space.
+
+#### What are you going to prepare to deliver the content?
```

Comment:
Wherever people want to! I don't want this to be a top-down thing where I go in having decided what's going to happen this time, and honestly I don't see myself as a "content-deliverer" too much — this is far more about giving people the opportunity to learn from each other than from making them learn from me!